app/views/posts/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Posts</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Body</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @posts.each do |post| %>
+      <tr>
+        <td><%= post.title %></td>
+        <td><%= post.body %></td>
+        <% if user_signed_in? %>
+          <td><%= post.user.username %></td>
+        <% end %>
```

Comment:
- [ ] put this if statement in a helper method and render its content and that you call here instead of having the if statement directly in the view

Reply:
Done